### PR TITLE
More compat data for WindowControlsOverlay

### PR DESCRIPTION
**Pull request by captainbrosset**
#### Summary
The Window Controls Overlay feature is actually available starting with Chrome and Edge 98 (which is when it's out of origin trial). We had originally documented 97, and I'm updating the version here.

I'm also adding the data for the WindowControlsOverlay JS interface. The reason for this is that I'm writing some reference documentation for it in https://github.com/mdn/content/pull/11368.

#### Related issues
See https://github.com/mdn/content/pull/11368 and https://github.com/mdn/content/issues/10539 for more context on this change.

### CODE REVIEW

File: api/WindowControlsOverlay.json [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "geometrychange_event": {
```
Comment (ddbeck) [inline api/WindowControlsOverlay.json:51]: Is this right? The linked spec seems to say the event is "change" (implying `change_event` here). I haven't actually used the API though, so it's possible the actual implementation is different

File: api/WindowControlsOverlayGeometryChangeEvent.json [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
```
Comment (ddbeck) [inline api/WindowControlsOverlayGeometryChangeEvent.json:95]: For consistency with the parent feature

```suggestion
            "experimental": true,
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
Thanks for the PR. A few suggestions for your, @captainbrosset.

Also, for posterity, would you mind linking to a Chrome Status entry or bug or something that shows that this shipped in the version noted? Nice to have when revisiting this data in the future. Thank you again!

File: api/WindowControlsOverlay.json [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+          "webview_android": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
```
[the remark this answers is not in the document]
Comment (captainbrosset) [reply api/WindowControlsOverlay.json:50]: No, you can access the only instance via navigator.windowControlsOverlay, but can't construct new instances.

File: api/WindowControlsOverlay.json [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+      "geometrychange_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WindowControlsOverlay/ongeometrychange",
```
[the remark this answers is not in the document]
Comment (captainbrosset) [reply api/WindowControlsOverlay.json:53]: Ah, thanks. Indeed, I changed this in my mdn/content PR, but forgot to change the URL here. Doing it now.

File: api/WindowControlsOverlay.json (hunk 1 above)
Comment (captainbrosset) [reply api/WindowControlsOverlay.json:51]: Yeah, there was the same confusion with the event name on the MDN PR. But testing live in chromium shows that the event name is geometrychange. So I think we should keep it like it is here.

File: api/WindowControlsOverlayGeometryChangeEvent.json (hunk 2 above)
Comment (captainbrosset) [reply api/WindowControlsOverlayGeometryChangeEvent.json:95]: Done

**Comment by captainbrosset** [comment]
Here is the chromestatus.com feature entry: https://chromestatus.com/feature/5741247866077184

**Comment by captainbrosset** [comment]
@ddbeck thanks for the review. I think I addressed all of your comments.

File: api/WindowControlsOverlay.json (hunk 1 above)
Comment (ddbeck) [reply api/WindowControlsOverlay.json:51]: Agreed.

**Review by ddbeck — APPROVED** [review]
Looks great. Thank you! 🎉

**Pull request merged**